modules/compiler/targets/host/source/target.cpp

```diff
@@ -180,6 +231,24 @@ compiler::Result HostTarget::initWithBuiltins(
     }
   } else if (triple.isX86()) {
     CPUName = "x86-64-v3";  // Default only, may be overridden below.
+  } else if (triple.isRISCV()) {
+    // These are reasonable defaults, which has been used for various RISC-V
+    // target so far. We should allow overriding of this in the future
```

Comment:
nit: I think the CPU is overrideable, through `CA_HOST_TARGET_CPU`. Unless you mean the ABI?